pycbc/io/hdf.py

```diff
@@ -822,14 +870,18 @@ def to_coinc_xml_object(self, file_name):
             # FIXME: As two-ifo is hardcoded loop over all ifos
             sngl_combined_mchirp = 0
             sngl_combined_mtot = 0
+            net_snrsq = 0
             for ifo in ifos:
-                sngl_id = self.trig_id[ifo][idx]
-                event_id = lsctables.SnglInspiralID(sngl_id)
+                if not sngl_col_vals['snr'][ifo][1][idx]:
+                    continue
```

Comment:
`['snr']` -> SNR values
`['ifo']` -> Which ifo is in play
`[1]` -> At this level we have a couple, the values and the logic array showing what is valid. So we take the logic array
`[idx]` -> The index of the point we are looking at.

In short we're testing if this ifo is participating in this coincidence.

Reply:
Ah okay, makes sense. Maybe if you put this as a variable called ifo_participating and it will be more obvious i.e. "`if not ifo_participating`" does what it says on the tin